lib/prometheus/active_record.rb

```diff
@@ -2,18 +2,17 @@
 
 Yabeda.configure do
   group :rails_connection_pool do
-    size = gauge :size, comment: 'Size of the connection pool'
-    connections = gauge :connections, comment: 'Number of connections in the connection pool'
-    waiting = gauge :waiting, comment: 'Number of waiting in the queue of the connection pool'
-
-    no_labels = {}.freeze
+    no_labels = { state: nil }.freeze
```

Comment:
This is because you can't set different labels (empty and other with `:state`) to the same gauge
A gauge needs to consistently set the same labels